### PR TITLE
Add more links, tweak structure and wording

### PR DESCRIPTION
**Pull request by jonator**
I made wording a bit more formal and consistent.

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,29 +1,30 @@
-## Osmosis-Labs
+# Osmosis Labs
 
-The Osmosis-labs hosts a series of repositories relating to the Osmosis blockchain and tooling for Cosmos.
+Repositories for the Osmosis AMM and tooling for Cosmos.
```
Comment (ValarDragon) [inline profile/README.md]: I think the prior sentence is more accurate.

There are other repositories for the osmosis blockchain not in this org, and its for the blockchain, not just the AMM!

File: profile/README.md [hunk 2]
```diff
@@ -1,29 +1,30 @@
...
 
-* [osmosis](https://github.com/osmosis-labs/osmosis) - A node implementation for the Osmosis chain
-* [cosmos-sdk fork](https://github.com/osmosis-labs/cosmos-sdk) - Fork of the Cosmos SDK used in osmosis
+- [osmosis](https://github.com/osmosis-labs/osmosis) - A node implementation for the Osmosis chain, built with a [fork](https://github.com/osmosis-labs/cosmos-sdk) of the [Cosmos SDK](https://github.com/cosmos/cosmos-sdk).
+- [LocalOsmosis](https://github.com/osmosis-labs/LocalOsmosis) - One-click local Osmosis testnet and ecosystem for rapid prototyping. Inspired by LocalTerra.
 
 Go to https://get.osmosis.zone to quickstart setting up a node!
 
-## Cosmwasm ecosystem
+### Cosmwasm ecosystem
```
Comment (ValarDragon) [inline profile/README.md]:
```suggestion
## Cosmwasm ecosystem
```

We can put this below frontend tho

**Comment by ValarDragon** [comment]
Looks great to me except first sentence change, nice!!

**Comment by jonator** [comment]
I'm AFK for the week so feel free to make any desired changes and merge @ValarDragon

**Comment by jonator** [comment]
~~@ValarDragon Think we could get this in?~~

Oops, forgot there were requested changes. Done.

**Pull request closed without merging**